docs/user_guide/write_api/push_job.md

```diff
@@ -27,6 +27,14 @@ When performing an Incremental Push, no new dataset versions are created, and da
 of the dataset. This leverages the same mechanism as Streaming Writes, and requires that the store be configured as 
 Hybrid.
 
+### Targeted Region Push
+Technically, targeted region push is an option of full push _(hence not a new mode)_, but it allows writing data into a 
```

Comment:
Are we not supporting target colo with inc_push ?

Reply:
We don't support it now so better just not to mention it?